qgis-app/plugins/views.py

```diff
@@ -422,17 +423,22 @@ def get_paginate_by(self, queryset):
     def get_queryset(self):
         qs = super(PluginsList, self).get_queryset()
         sort_by = self.request.GET.get('sort', None)
+        chars_fields = ['name', 'author']
         if sort_by:
             if sort_by[0] == '-':
                 _sort_by = sort_by[1:]
             else:
                 _sort_by = sort_by
-            try:
-                # Average vote is not a field!
-                _sort_by == 'average_vote' or self.model._meta.get_field(_sort_by)
+
```

Comment:
I'd stick to the try/except or we'll get ton of errors, this parameter comes straight from the query string.